server/routes/pwd.route.js

```diff
@@ -0,0 +1,12 @@
+const express = require('express');
+const {
+  forgotPwdController,
+  resetPasswordController,
+} = require('../controllers/pwd.controllers');
+
+const router = express.Router();
+
+router.get('/_reset_password/:JWT_TOKEN', resetPasswordController);
+router.post('/_forgot_password', forgotPwdController);
```

Comment:
This route should be authenticate, you can look ```authMiddleware```

Reply:
if the user is authenticated, they would obviously knew the password and does not require _reset_password right?i mean they can't access the route right? 🤔, if i am missing something please inform me the flow, and what  role authmiddleware plays here, thanks

Reply:
Yah ,sorry my mistake :/